lib/Rx/Observable.php

```diff
@@ -839,4 +840,19 @@ public function timeout($timeout, ObservableInterface $timeoutObservable = null,
             return new TimeoutOperator($timeout, $timeoutObservable, $scheduler);
         });
     }
+
+    /**
+     * Projects each element of an observable sequence into zero or more buffers which are produced based on
+     * element count information.
+     *
+     * @param $count
+     * @param int $skip
+     * @return AnonymousObservable
+     */
+    public function bufferWithCount($count, $skip = -1)
```

Comment:
Looked at the tests. `$skip` is cool. Let's keep it, but signaling "no skip" with `null` instead of `-1` seems like a good idea to me. What do you think?